src/Compilers/Core/Portable/Emit/SemanticEdit.cs

```diff
@@ -31,6 +31,12 @@ namespace Microsoft.CodeAnalysis.Emit
         /// </summary>
         public ISymbol? NewSymbol { get; }
 
+        /// <summary>
+        /// The symbol that matches the containing symbol of OldSymbol, but from
+        /// the later compilation. Used to turn deletes into updates.
+        /// </summary>
+        public ISymbol? NewContainingSymbol { get; }
```

Comment:
>NewContainingSymbol [](http://example.com/codeflow?start=24&length=19)

I think it'd be fine to reuse `NewSymbol` and document that for Delete operation it's the container.